### DISCUSSION TITLE
Article about Django and REST APIs

### DISCUSSION THREAD

**Discussion by AbhijithGanesh** [EddieHubCommunity/support#2287] [Show and tell]
I have written an article about Django and REST APIs, feel free to read it.
https://dev.to/abhijithganesh/what-does-it-take-to-build-an-unconventional-rest-api-in-python-3oog

**Comment by ashwinexe** [comment]
Hi, Thanks for sharing this. Unfortunately I don't think this article was aimed for beginners to understand what Django REST Framework is and how to get started with it (since I am a beginner here). Hope you understand :)

**Reply by AbhijithGanesh** [reply]
Yes I got it, this isn't meant for beginners. It was just to show that there are other options out which are equally good and even better than DRF. It might be of use once you understand and assimilate how REST-APIs are implemented in Django.
